src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,9 @@
+$icons-size: 24px;
+$c-white: #fff;
+$c-font: #1b2129;
```

Comment:
Do not abbreviate the names to unreadable ones and try not to use names based on the content of the variable, choose universal names based on their meaning.